cypher/src/main/java/org/vertexium/cypher/ast/model/CypherExpression.java

```diff
@@ -9,6 +9,7 @@
 public abstract class CypherExpression extends CypherAstBase {
     public static String toString(Iterable<CypherAstBase> expressions) {
         return stream(expressions)
+                .map(e -> e == null ? "null" : "")
```

Comment:
Should this maybe be `.map(e -> e == null ? "null" : e)`?

Reply:
fixed